osu.Game/Screens/Ranking/Expanded/Accuracy/RankNotch.cs

```diff
@@ -41,7 +41,7 @@ private void load()
                     Origin = Anchor.TopCentre,
                     RelativeSizeAxes = Axes.Y,
                     Height = AccuracyCircle.RANK_CIRCLE_RADIUS,
-                    Width = 1f,
+                    Width = (float)AccuracyCircle.NOTCH_WIDTH_PERCENTAGE * 360f,
```

Comment:
What's this for?

Reply:
To match [the check](https://github.com/ppy/osu/pull/22554/files#diff-6ad7f21c9f785ced34f1b5dea121ccf10e8ff29baae1b2ec9b08a30b4bae6cd3R297) for determining if the accuracy gauge lands in the gap (this is that gap)

It very closely matches what we had before. With the current `NOTCH_WIDTH_PERCENTAGE` this becomes 0.72, but this can be increased if desired, see the comparison videos in OP.